src/components/modal/v2/styles/components/_calculator.scss

```
@@ -34,110 +34,119 @@
-            @include lander {
-                margin-top: 20px;
+            @include desktop {
+                @include lander {
+                    margin-top: 20px;
+                }
             }
         }
-    }
-
-    .input__wrapper {
-        position: relative;
-        display: inline-block;
-        font-size: 0.9rem;
-        text-align: left;
-        margin-bottom: 0.25rem;
-
-        .input__label {
-            position: absolute;
-            left: 0.5rem;
-            top: 0.25rem;
-        }
 
-        .input__currency-symbol {
-            position: absolute;
-            right: 0;
-            margin-top: 25px;
-            font-size: 18px;
-            margin-right: 10px;
-        }
+        .input__wrapper {
+            position: relative;
+            display: inline-block;
+            font-size: 0.9rem;
+            text-align: left;
+            margin-bottom: 0.25rem;
 
-        .input {
-            font-family: $paypal-open;
-            background: white;
-            border: 1px solid #ccc;
-            border-radius: 3px;
-            color: $text-main;
-            font-size: 18px;
-            padding: 1.75rem 0.5rem 0.25rem;
-
-            &.empty-input {
-                padding-top: 16px;
-                padding-bottom: 16px;
-                &::placeholder {
-                    font-family: $paypal-open;
-                    font-size: 14px;
-                }
+            .input__label {
+                position: absolute;
+                left: 0.5rem;
+                top: 0.25rem;
             }
-        }
-    }
 
-    .form {
-        text-align: left;
+            .input__currency-symbol {
+                position: absolute;
+                right: 0;
+                margin-top: 25px;
+                font-size: 18px;
+                margin-right: 10px;
+            }
 
-        @include tablet {
-            // text-align: center;
+            .input {
+                font-family: $paypal-open;
+                background: white;
+                border: 1px solid #ccc;
+                border-radius: 3px;
+                color: $text-main;
+                font-size: 18px;
+                padding: 1.75rem 0.5rem 0.25rem;
+
+                &.empty-input {
+                    padding-top: 16px;
+                    padding-bottom: 16px;
+                    &::placeholder {
+                        font-family: $paypal-open;
+                        font-size: 14px;
+                    }
+                }
+            }
         }
-    }
 
-    .v4Design {
-        h4 {
-            font-weight: 700;
+        &.v4Design, &.checkout {
+            h4 {
+                font-weight: 700;
+            }
         }
-    }
 
-    &__error {
-        color: $text-main;
-        font-size: 14px;
-        margin-top: 10px;
+        &.checkout {
+            .input__wrapper {
+                // @include mobile {
+                    width: 100%;
+                // }
```

```suggestion
            .input__wrapper {
                    width: 100%;
```
You should be safe to remove the commented out mixin here and below on `.input`.